cypress/e2e/signIn.cy.js

```diff
@@ -1,11 +1,69 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
+  const username = 'tomsmith';
+  const password = 'SuperSecretPassword!';
+
   beforeEach(() => {
+    cy.visit('https://the-internet.herokuapp.com/login');
+  });
+
+  it('should provide an ability to log in with valid credentials', () => {
+    cy.get('h2')
+      .should('contain.text', 'Login Page');
+
+    cy.get('#username').type(username);
+    cy.get('#password').type(password);
+    cy.get('.radius').click();
+    cy.contains('#flash', 'You logged into a secure area!')
+      .should('exist');
+  });
+
+  it(`should display validation error when entering invalid 'username'`, () => {
+    cy.get('h2')
+      .should('contain.text', 'Login Page');
+
+    cy.get('#username').type(username + '@');
```

Comment:
```suggestion
    cy.get('#username').type(username + '@');
```
create pls invalidUsername variable for this value, because it is not very readable